Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -180,6 +180,7 @@ WI.loaded = function()
     WI.modifierKeys = {altKey: false, metaKey: false, shiftKey: false};
     WI.visible = false;
     WI._windowKeydownListeners = [];
+    WI.dropActive = false;
```

Comment:
rather than declaring this globally, how about having this be `WI.DropZoneView._active` so it's a bit more contained (and since it's only used by `WI.DropZoneView`)?

another option would be to do something similar to how `WI.ImageResourceContentView` has `disableInteractions` and propagate that when viewing the Network Tab.  tho this might be overly preventative (i.e. we may want to allow developers to drag over the `WI.ImageResourceContentView` to create a response local override)

i suggest the above because i think the only way that the `WI.DropZoneView` inside `WI.ImageResourceContentView` can get activated is if the user somehow starts their drag inside the `WI.ImageResourceContentView` instead of the `WI.NetworkTabContentView` (e.g. have a Finder window perfectly positioned to do so), and i dont think we want that kind of inconsistency